File: 02-bit-magic/01-decimal-to-binary/01-decimal-to-binary.ts
const decimal_to_binary = (num: number) => {
  let res = ''
  while (num > 0) {
    res = `${num % 2}` + res
    num = Math.floor(num / 2)
  }
  return res
}

console.log(decimal_to_binary(7))

File: 02-bit-magic/01-decimal-to-binary/01-decimal-to-binary.py

def decimal_to_binary(dec):
  binary = ""
  quotient = dec
  while quotient:
    reminder = quotient % 2
    quotient = quotient // 2
    binary = str(reminder) + binary
  
  return binary

res = decimal_to_binary(18)
print(res)

res = decimal_to_binary(12)
print(res)
